examples/02.deposit.js

```diff
@@ -26,7 +26,7 @@ const dvfConfig = {
 ;(async () => {
   const dvf = await DVF(web3, dvfConfig)
 
-  const depositResponse = await dvf.deposit('ETH', 0.45, starkPrivKey)
+  const depositResponse = await dvf.deposit('ETH', 0.70, starkPrivKey)
```

Comment:
Would prefer we keep this example smaller (since we only get 1 ETH from faucet). So with smaller amounts the example works multiple times